packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -126,25 +130,30 @@ export class SOQLEditorInstance {
     webviewPanel.onDidDispose(this.dispose, this, this.subscriptions);
   }
 
+  protected sendMessageToUi(
+    type: string,
+    payload?: string | string[] | DescribeSObjectResult
+  ): void {
+    this.webviewPanel.webview
+      .postMessage({
+        type,
+        payload
+      })
+      .then(undefined, (err: string) => {
+        showAndTrackError(type, err);
```

Comment:
what would cause this error to happen? I don't think these error messages should appear as toast notification for customers as they would be very annoying and not necessarily useful. We may be better off logging such issues in console.

Reply:
OK.  I'll move away from the toast message.  However, i did wonder if the getting the lists of objects in the org ( or the metadata for a single object ) should throw a toast up?  Luis shows some screen shots of those in the PR and I'm working with Emily to provide a user friendly error on those.

Reply:
I really do not know how this error might show up.  It could be a couple issues more related to app configuration and would most likely be caught during development.  

1.  The Content Security Policy is not set correctly on the webview.  
2. There was an error generating the webview and it cannot be messaged by vscode.